pkg/security/secl/model/accessors_windows.go

```diff
@@ -1,7 +1,10 @@
+//go:build windows
```

Comment:
gh won't allow me to expand it; this comment is from the previous file (accessors_other.go)
That file does not have the `// Code generated - DO NOT EDIT.` comment, and the build tags are in the right order.  So is this file (still) not generated?  It's manually created?

Reply:
Yes this file is manually created until you manage to generate it yourself